x/accounts/msg_server.go

```diff
@@ -0,0 +1,105 @@
+package accounts
+
+import (
+	"context"
+
+	v1 "cosmossdk.io/x/accounts/v1"
+)
+
+var _ v1.MsgServer = msgServer{}
+
+func NewMsgServer(k Keeper) v1.MsgServer {
+	return &msgServer{k}
+}
+
+type msgServer struct {
+	k Keeper
+}
+
+func (m msgServer) Create(ctx context.Context, request *v1.MsgCreate) (*v1.MsgCreateResponse, error) {
```

Comment:
should we call this init to align instead of having init and create

Reply:
done!